JuliaGPU/session2/Aniruddh Patil/q1.jl

```diff
@@ -0,0 +1,55 @@
+#Sorting Algorithm Benchmark (Random Pivot Quick Sort vs. In-built Sort)
+using BenchmarkTools
+function quicksort(A,i=1,j=length(A))
```

Comment:
quicksort is a mutating function since it modifies contents of A. By convention, mutating functions are terminated with '!'.